api/lecture/config/routes.json

```diff
@@ -34,7 +34,7 @@
     },
     {
       "method": "PUT",
-      "path": "/lectures/:id",
+      "path": "/lectures",
```

Comment:
@alaa-yahia  .. can we leave the default PUT /lecture/:id, and create a new one just for sorting lectures. Can be something like `PUT /lectures/sort-order` or `PUT /lectures/bulk-update` (you also need to update permissions to allow only authenticated).

We will need `/lectures/:id` soon to update things related to a single lecture like title and such.